### ISSUE TITLE
Removing <code> tags from Accordions

### ISSUE THREAD

**Issue by bwaltz** [ColbyCommunications/colby-college-theme#64]
@gregoryspragginsjr We're finding that lists in Accordions are getting styles from the `<code>` element that is surrounding things:

![Screen Shot 2022-12-08 at 2 09 24 PM](https://user-images.githubusercontent.com/2333967/206545860-42f6cfcf-5215-4599-9b0c-69cec4c538d9.png)

You can see the lists inside are displaying as monospaced.

Can we remove the `<code>` elements? https://github.com/ColbyCommunications/colby-college-theme/blob/master/src/twig/components/accordion/accordion.twig#L48

When I try to delete them it screws everything up on my local:

![Screen Shot 2022-12-08 at 2 07 19 PM](https://user-images.githubusercontent.com/2333967/206546135-8c8bd6ef-dac5-4737-8eb9-34efb8b40d69.png)

![Screen Shot 2022-12-08 at 2 12 29 PM](https://user-images.githubusercontent.com/2333967/206546225-b646b862-1817-4498-a973-81e92c2d4dc0.png)

**Comment by gregoryspragginsjr** [comment]
@bwaltz I can take a look here. The `<code>` tag is required here to prevent the error message your seeing (encoding error). There should be a workaround.

**Comment by bwaltz** [comment]
@gregoryspragginsjr checking in on this one. Do you have any thoughts?

**Comment by gregoryspragginsjr** [comment]
Hey @bwaltz sorry about the delay on these; been on sick leave. You should see an update for this and a few things in the next day or so. The `code` tag unfortunately is necessary in this context as you can see from removing it. It'll just need some additional styles.

**Comment by gregoryspragginsjr** [comment]
Okay @bwaltz I've applied a small change here that should remedy this issue. I did notice that you guys have made a fair number of changes, migrating some of the tailwind utility classes in the wysiwyg twig file into scss and utilizing `apply`. The only way to to remedy this issue in particular regarding the necessary `code` tag stylings is to apply some type of specificity to override via either:

1. `>` applications to direct children
2. `!important` overrides to all styles within your set `ul` on line 86 in `styles.scss`
3. Add another classname for specificity

I've applied the first approach to your css but feel free to change this.
